.expeditor/update_version.sh

```diff
@@ -7,6 +7,9 @@
 set -evx
 
 sed -i -r "s/^(\s*)VERSION = \".+\"/\1VERSION = \"$(cat VERSION)\"/" components/ruby/lib/license_acceptance/version.rb
+cd components/ruby
+bundle install
```

Comment:
Shouldn't this be a bundle install / bundle update?

Reply:
I was conflicted on that. My initial goal was to just get the `license-acceptance` version aligned. If I `bundle update` it will update all the dependencies... which I realize now is completely fine. Update incoming!